crates/aiken-project/Cargo.toml

```diff
@@ -20,6 +20,7 @@ ignore = "0.4.20"
 indexmap = "1.9.2"
 itertools = "0.10.5"
 miette = { version = "5.5.0", features = ["fancy"] }
+minicbor = "0.19.1"
```

Comment:
It's not a problem but we can use the re-exported minicbor from pallas whenever we need it. I wouldn't worry about switching this, it's just good to know.